src/components/ServiceList.js

```diff
@@ -1,30 +1,39 @@
-import React, {Component} from "react";
-import {connect} from "react-redux";
-import PropTypes from "prop-types";
+import React from "react";
+import { useSelector } from "react-redux";
 
-import {Link} from "react-router-dom";
+import { Link } from "react-router-dom";
 
-import {Table, Typography, Icon, Tag} from "antd";
+import { Table, Typography, Icon, Tag } from "antd";
 
-import {chordServicePropTypesMixin, serviceInfoPropTypesShape} from "../propTypes";
-import {ROLE_OWNER} from "../constants";
-import {withBasePath} from "../utils/url";
+import { ROLE_OWNER } from "../constants";
+import { withBasePath } from "../utils/url";
 
-const ARTIFACT_STYLING = {fontFamily: "monospace"};
+const ARTIFACT_STYLING = { fontFamily: "monospace" };
 
 // biggest reasonable size limit before rolling over
 // currently 11 services including gohan
 const MAX_TABLE_PAGE_SIZE = 12;
 
-const serviceColumns = isOwner => [
+const iconStyle = {
+    fontSize: "20px",
+};
+
+const serviceColumns = (isOwner) => [
     {
         title: "Artifact",
         dataIndex: "type.artifact",
-        render: artifact => artifact
-            ? (isOwner
-                ? <Link style={ARTIFACT_STYLING} to={withBasePath(`admin/services/${artifact}`)}>{artifact}</Link>
-                : <span style={ARTIFACT_STYLING}>{artifact}</span>)
-            : ""
+        render: (artifact) =>
+            artifact ? (
+                isOwner ? (
+                    <Link style={ARTIFACT_STYLING} to={withBasePath(`admin/services/${artifact}`)}>
+                        {artifact}
+                    </Link>
+                ) : (
+                    <span style={ARTIFACT_STYLING}>{artifact}</span>
+                )
+            ) : (
+                ""
+            ),
```

Comment:
I have made a change, not exactly what you asked for, just cause it would be really plausible that in the future I might unknowingly end up introducing that change again. I do feel that the ending part is not great so changed that